<launch>

  <!-- ROS-Pybullet interface -->
  <node pkg="ros_pybullet_interface" name="ros_pybullet_interface" type="ros_pybullet_interface_node.py" output="screen">
    <rosparam param="config" file="$(find rpbi_examples)/configs/pybullet_objects_example/config.yaml"/>
  </node>

  <!-- Static transforms -->
  <node pkg="rpbi_examples" type="orbit_node.py" name="camera_orbit_tf" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="figure_eight_base_in_rpbi_world" args="0.5 0 0.65 0 1.57 0 rpbi/world figure_eight_base"/>

  <!-- Generate fig8 motion for visual object -->
  <node pkg="ik_ros_examples" name="fig_eight" type="figure_eight_node.py"/>

  <!-- Run example node -->
  <node pkg="rpbi_examples" name="example" type="pybullet_objects_example_node.py" output="screen"/>

  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find rpbi_examples)/configs/rgbd.rviz" />

</launch>
